src/com/example/data.cljs

```diff
@@ -0,0 +1,66 @@
+(ns com.example.data)
+
+(def readiness-data
+  #:quiz{:version 0
+         :id "375623e0-92dd-4815-b507-4d577a55f37c"
+         :label "Readiness Quiz"
+         :questions
+         [#:question{:label "How many days in a row have you trained?"
+                     :id "trained",
+                     :options
+                     [#:option{:label "4+ days", :value 1}
+                      #:option{:label "3 days", :value 2}
+                      #:option{:label "2 days", :value 3}
+                      #:option{:label "1 days", :value 4}
```

Comment:
I updated this tree of data with namespaced keywords.

Reply:
This makes it so we can pass this object straight to our pathom resolver and not have to massage this data into what the resolver is looking for.

Reply:
where do you submit the form?